tests/dmflakey/dmflakey.go

```diff
@@ -0,0 +1,315 @@
+//go:build linux
+
+package dmflakey
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"golang.org/x/sys/unix"
+)
+
+type featCfg struct {
+	// SyncFS attempts to synchronize filesystem before inject failure.
+	syncFS bool
+	// interval is used to determine how long the failure lasts.
+	interval time.Duration
+}
+
+// Default values.
+var (
+	defaultImgSize  int64 = 1024 * 1024 * 1024 * 10 // 10 GiB
+	defaultInterval       = 2 * time.Minute
```

Comment:
Please define the two items as constants.

Reply:
Sure. Updated